internal/apis/certmanager/types_certificate.go

```diff
@@ -30,162 +30,217 @@ import (
 // The stored certificate will be renewed before it expires (as configured by `spec.renewBefore`).
 type Certificate struct {
 	metav1.TypeMeta
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
 	metav1.ObjectMeta
 
-	// Desired state of the Certificate resource.
+	// Specification of the desired state of the Certificate resource.
+	// https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
 	Spec CertificateSpec
 
-	// Status of the Certificate. This is set and managed automatically.
+	// Status of the Certificate.
+	// This is set and managed automatically.
+	// Read-only.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
 	Status CertificateStatus
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// CertificateList is a list of Certificates
+// CertificateList is a list of Certificates.
 type CertificateList struct {
 	metav1.TypeMeta
+	// Standard list metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
 	metav1.ListMeta
 
-	Items []Certificate
+	// List of Certificates
+	Items []Certificate `json:"items"`
 }
 
 type PrivateKeyAlgorithm string
 
 const (
-	// Denotes the RSA private key type.
+	// RSA private key algorithm.
 	RSAKeyAlgorithm PrivateKeyAlgorithm = "RSA"
 
-	// Denotes the ECDSA private key type.
+	// ECDSA private key algorithm.
 	ECDSAKeyAlgorithm PrivateKeyAlgorithm = "ECDSA"
 
-	// Denotes the Ed25519 private key type.
+	// Ed25519 private key algorithm.
 	Ed25519KeyAlgorithm PrivateKeyAlgorithm = "Ed25519"
 )
 
 type PrivateKeyEncoding string
 
 const (
-	// PKCS1 key encoding will produce PEM files that include the type of
-	// private key as part of the PEM header, e.g. `BEGIN RSA PRIVATE KEY`.
-	// If the keyAlgorithm is set to 'ECDSA', this will produce private keys
-	// that use the `BEGIN EC PRIVATE KEY` header.
+	// PKCS1 private key encoding.
+	// PKCS1 produces a PEM block that contains the private key algorithm
+	// in the header and the private key in the body.
+	// It can be recognised by its `BEGIN RSA PRIVATE KEY` or `BEGIN EC PRIVATE KEY` header.
```

Comment:
**suggestion:** Please either keep the text wrapped within paragraphs, and no line break within a single paragraph. Keeping a consistent line length is important for readability. Two choices:
1. If you want to continue with the same paragraph, keep going on the same line, only break the line when the line needs to be wrapped.
2. If you want to start a new paragraph, leave an empty line to help the eye jump to the next paragraph.

If keeping the short descriptive line is important, then use a different paragraph:

```suggestion
	// PKCS #1 private key encoding.
	//
	// The PKCS #1 encoding produces a PEM block that include the type of
	// private key as part of the PEM header, e.g., `BEGIN RSA PRIVATE KEY`.
	// or `BEGIN EC PRIVATE KEY`.
```